lib/Services/JsonWebTokenBuilderService.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace SimpleSAML\Module\oidc\Services;
+
+use DateTimeImmutable;
+use Exception;
+use Lcobucci\JWT\Builder;
+use Lcobucci\JWT\Configuration;
+use Lcobucci\JWT\Encoding\ChainedFormatter;
+use Lcobucci\JWT\Signer;
+use Lcobucci\JWT\Signer\Key\InMemory;
+use Lcobucci\JWT\UnencryptedToken;
+use League\OAuth2\Server\Exception\OAuthServerException;
+use ReflectionException;
+use SimpleSAML\Module\oidc\Utils\FingerprintGenerator;
+use SimpleSAML\Module\oidc\Utils\UniqueIdentifierGenerator;
+
+class JsonWebTokenBuilderService
+{
+    protected ConfigurationService $configurationService;
+    protected Configuration $jwtConfig;
+
+    /**
+     * @throws ReflectionException
```

Comment:
missing @param for the constructor argument

Reply:
ok